bindata/etcd/pod.yaml

```diff
@@ -201,6 +201,12 @@ ${COMPUTED_ENV_VARS}
       failureThreshold: 3
       periodSeconds: 5
       successThreshold: 1
+    livenessProbe:
```

Comment:
```suggestion
    livenessProbe:
        httpGet:
          path: health
          port: 9978
          scheme: HTTPS
        timeoutSeconds: 10
        periodSeconds: 5
        successThreshold: 1
        failureThreshold: 3
```

Reply:
@Elbehery seems to work without the certs, magic :shrug:

Reply:
hahaha :D :D thx so much :D